### PR TITLE
Add support for ALIAS, CAA and SSHFP DNS records

### PR DESCRIPTION
**Pull request by Archirk**
- Add support for ALIAS, CAA and SSHFP dns records
- Increase dupl threshold in golangci to pass test

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ IMPROVEMENTS:
 
 * Added `nodegroup_type` attribute to `selectel_mks_nodegroup_v1` resource ([#202](https://github.com/selectel/terraform-provider-selectel/issues/202))
 * Added handling for private kube API clusters to `selectel_mks_cluster_v1` resource ([#204](https://github.com/selectel/terraform-provider-selectel/pull/204))
+* Added support for ALIAS, CAA and SSHFP dns records to `selectel_domains_record_v1` resource ([#210](https://github.com/selectel/terraform-provider-selectel/issues/210))
```
Comment (kolsean) [inline CHANGELOG.md:7]: Please do not edit changelog here. It will be updated by separate PR later.

**Review by kolsean — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (Archirk) [reply CHANGELOG.md]: @kolsean removed

**Review by kolsean — CHANGES_REQUESTED** [review]
Please also update the documentation for related objects at `/website/docs/r/`

**Review by kolsean — APPROVED** [review]

**Review by kolsean — CHANGES_REQUESTED** [review]
Need to fix acceptance tests:
```
TF_ACC=1 go test ./selectel -run TestAccDomains -timeout 360m
--- FAIL: TestAccDomainsRecordV1Basic (19.29s)
    resource_selectel_domains_record_v1_test.go:44: Step 2/2 error: Check failed: Check 47/59 error: selectel_domains_record_v1.record_caa_tf_acc_test_1: Attribute 'value' expected "letsencrypt-2.com", got "letsencrypt2.com"
FAIL
FAIL    github.com/terraform-providers/terraform-provider-selectel/selectel     33.569s
```

**Comment by Archirk** [comment]
> Need to fix acceptance tests:
> 
> ```
> TF_ACC=1 go test ./selectel -run TestAccDomains -timeout 360m
> --- FAIL: TestAccDomainsRecordV1Basic (19.29s)
>     resource_selectel_domains_record_v1_test.go:44: Step 2/2 error: Check failed: Check 47/59 error: selectel_domains_record_v1.record_caa_tf_acc_test_1: Attribute 'value' expected "letsencrypt-2.com", got "letsencrypt2.com"
> FAIL
> FAIL    github.com/terraform-providers/terraform-provider-selectel/selectel     33.569s
> ```

Fixes typo in records test. + gofmt-ed some files since linter was unsatisfied.
Could not run testacc succesfull since face some quote errors, despite having free quotes.
Errors are not related to my part: DBaas and Subnets
I need to know exact quotes since each testacc wastes more almost 3 hours (10451 sec as example)

File: selectel/mks.go [hunk 2]
```diff
@@ -57,7 +57,8 @@ func getMKSClusterV1Endpoint(region string) (endpoint string) {
 }
 
 func waitForMKSClusterV1ActiveState(
-	ctx context.Context, client *v1.ServiceClient, clusterID string, timeout time.Duration) error {
+	ctx context.Context, client *v1.ServiceClient, clusterID string, timeout time.Duration,
+) error {
```
Comment (kolsean) [inline selectel/mks.go:61]: We do not need this line break here and below. Please roll back these changes.

**Review by kolsean — CHANGES_REQUESTED** [review]

File: selectel/mks.go (hunk 2 above)
Comment (Archirk) [reply selectel/mks.go]: formating changes for mks and dbaas related files are reset

**Review by kolsean — APPROVED** [review]

**Pull request merged**